includes/fs-core-functions.php

```diff
@@ -118,7 +118,7 @@ function fs_enqueue_local_style( $handle, $path, $deps = array(), $ver = false,
     }
 
     if ( ! function_exists( 'fs_enqueue_local_script' ) ) {
-        function fs_enqueue_local_script( $handle, $path, $deps = array(), $ver = false, $in_footer = 'all' ) {
+        function fs_enqueue_local_script( $handle, $path, $deps = array(), $ver = false, $in_footer = false ) {

```

Comment:
@bahiirwa let's change the `$in_footer` default value to `true`. This way we won't be doing any breaking change.